fairing/train.py

```diff
@@ -108,19 +108,25 @@ def signal_handler(signal, frame):
 
         logger.warn("Training(s) launched.")
 
-        if stream_logs:
-            self.backend.stream_logs(self.image_name, self.image_tag)
+        if self.stream_logs:
+            self.backend.stream_logs(self.image_name, self.image_tag, self.namespace)
 
-    def start_training(self, user_class):
-        self.strategy.exec_user_code(user_class)
+        if self.cleanup:
+            self.backend.cleanup(self.image_name, self.image_tag, self.namespace)
+
+    def start_training(self, user_class, *args, **kwargs):
+        logger.warn("Starting user code!!!")
```

Comment:
Remove this (Or alternatively make it a more formal message and only in `debug`)

Reply:
will remove